config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -68,6 +68,8 @@
          files="src[\\/]test[\\/]resources[\\/].*[\\/]ant[\\/]checkstyleanttask[\\/]ExpectedCheckstyleAntTaskXmlOutput\.xml"/>
   <suppress id="lineLength"
          files="src[\\/]test[\\/]resources[\\/].*[\\/]checks[\\/]translation[\\/]Expected.*\.xml"/>
+  <suppress id="lineLength"
+            files="src[\\/]test[\\/]resources[\\/].*[\\/]meta[\\/].*"/>
```

Comment:
please share violations, it is not not good to suppress, and this files are not "expected" files.

Reply:
I have added comments why it is needed.

Reply:
we do validation of length in main code also, but by different Check 
https://github.com/checkstyle/checkstyle/blob/518f92c08f0e7bf85d82199003784827b3f9663b/config/checkstyle_checks.xml#L181-L184

please apply patch:
```
✔ ~/java/github/romani/checkstyle [gaurabdg/improve-metadata-UT L|✚ 2] 
05:14 $ git diff
diff --git a/config/checkstyle_non_main_files_checks.xml b/config/checkstyle_non_main_files_checks.xml
index 9fce9ad..a121f55 100644
--- a/config/checkstyle_non_main_files_checks.xml
+++ b/config/checkstyle_non_main_files_checks.xml
@@ -19,7 +19,7 @@
     <property name="id" value="lineLength"/>
     <!-- catch lines above 100 symbols -->
     <property name="format"
-             value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
+             value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd| \* \{@code| \* com\.)).{101,}$"/>
     <property name="message" value="Line should not be longer than 100 symbols"/>
   </module>
 
diff --git a/config/checkstyle_non_main_files_suppressions.xml b/config/checkstyle_non_main_files_suppressions.xml
index 51003e5..f67bb1f 100644
--- a/config/checkstyle_non_main_files_suppressions.xml
+++ b/config/checkstyle_non_main_files_suppressions.xml
@@ -68,10 +68,6 @@
          files="src[\\/]test[\\/]resources[\\/].*[\\/]ant[\\/]checkstyleanttask[\\/]ExpectedCheckstyleAntTaskXmlOutput\.xml"/>
   <suppress id="lineLength"
          files="src[\\/]test[\\/]resources[\\/].*[\\/]checks[\\/]translation[\\/]Expected.*\.xml"/>
-  <!-- we cannot suppress JavadocScraper test inputs' line lengths at will break code logic.
-       Module javadocs in main src do not have lineLegth violations -->
-  <suppress id="lineLength"
-            files="src[\\/]test[\\/]resources[\\/].*[\\/]meta[\\/].*"/>
 
   <!-- content is generated -->
   <suppress id="lineLength"

```